configure.ac

```diff
@@ -630,13 +622,8 @@ fi
 
 AC_CHECK_FUNCS([mblen], [], [HAVE_MBLEN=1], [])
 
-rm -f Makefile
-if test "$enable_maintainer_mode" = yes ; then
-	AC_MSG_NOTICE(creating maintainer Makefile)
-	ln -s maintainer.mak Makefile
-else
-	AC_CONFIG_FILES(Makefile ctags.1)
-fi
+AC_CONFIG_FILES([Makefile])
+AC_CONFIG_FILES([ctags.1])
```

Comment:
Thank you for your understanding of my odd reason.

Due to my intension is not to bring wonder to other persons,
I will fix.